src/dbnode/storage/limits/query_limits.go

```diff
@@ -22,17 +22,20 @@ package limits
 
 import (
 	"fmt"
+	"sync"
 	"time"
 
-	xerrors "github.com/m3db/m3/src/x/errors"
-	"github.com/m3db/m3/src/x/instrument"
-
 	"github.com/uber-go/tally"
 	"go.uber.org/atomic"
+	"go.uber.org/zap"
+
+	xerrors "github.com/m3db/m3/src/x/errors"
```

Comment:
nit; 2nd import group for these

Reply:
Lint complains at me for this

Reply:
https://buildkite.com/uberopensource/m3-monorepo-public/builds/6527#882c346c-a630-4813-94f2-19360e73b575